src/utils/TriangleUtilities.js

```diff
@@ -86,11 +86,12 @@ export function getTriangleHitPointInfo( point, geometry, triangleIndex, target
 	// find the associated material index
 	let materialIndex = 0;
 	const groups = geometry.groups;
+	const firstVertexIndex = triangleIndex * 3;
```

Comment:
This constant ```firstVertexIndex``` can be substituted with the previous constant ```a```.

Reply:
This `a`?

```js
const a = indices[ triangleIndex * 3 ];
```

It's set to a different value (the index into the vertex position buffer rather than into the index buffer). Or maybe I'm misunderstanding.

Reply:
Oh, you are right. sorry.